Enterprise-CMCS/managed-care-review
# MCR-4591 approval dashboards

## Summary

As a CMS and state user, 
I want to see which submissions have been approved on my dashboard 
so that I can understand at-a-glance the status of all submissions. 

AC: 

1. If a submission has been marked as "approved" 
Then when I navigate to the CMS and state submission dashboards I see the approved submissions have a status of "Approved" (see [Figma](https://www.figma.com/design/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?node-id=15083-83972&node-type=frame&t=M2hfGhPBmgAaylhF-0))

Sort order - the date the approval supersedes the submitted date

Badge color changes follow t[he newest designs](https://www.figma.com/design/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?node-id=15987-108059&node-type=frame&t=FmrhCVoW5u3ZZZSJ-0) (notice there are differences between the CMS and State user experiences)

### To Dos

- [x] Create a synthesized status for reviewStatus and status
- [x] Update badge colors to match new designs (difference in color between state and CMS dashboards)

#### Related issues

https://jiraent.cms.gov/browse/MCR-4591

#### Screenshots

<img width="1105" alt="Screenshot 2024-11-19 at 4 06 18 PM" src="https://github.com/user-attachments/assets/6e75a8b8-9c1d-41c3-ae59-47273cf39450">



#### Test cases covered

CMS dashboard - approved submissions appears in dashboard

State dashboard - approved submissions appears in dashboard

## QA guidance

- Sign in as a CMS user and approve a submission
- Should see the approved submission in the CMS dashboard. 
- Sign in as a state user and should see the approved submission in the state dashboard
